cluster/examples/workloads/kubernetes/wordpress/aws/provider.yaml

```diff
@@ -19,43 +19,48 @@ spec:
     name: aws-provider-creds
   region: REGION
 ---
-apiVersion: core.crossplane.io/v1alpha1
-kind: ResourceClass
+apiVersion: database.aws.crossplane.io/v1alpha1
+kind: RDSInstanceClass
 metadata:
   name: standard-mysql
   namespace: crossplane-system
-parameters:
+specTemplate:
   class: db.t2.small
   masterUsername: masteruser
-  securityGroups: RDS_SECURITY_GROUP #sg-1,sg-2
+  securityGroups:
+   - RDS_SECURITY_GROUP # sg-1
   subnetGroupName: RDS_SUBNET_GROUP_NAME # eks-db-subnets
-  size: "20"
-provisioner: rdsinstance.database.aws.crossplane.io/v1alpha1
-providerRef:
-  name: aws-provider
-  namespace: crossplane-system
-reclaimPolicy: Delete
+  size: 20
+  engine: mysql
+  providerRef:
```

Comment:
is this indentation by purpose?

Reply:
Yep! These fields are all part of the `specTemplate`